en/contribute/support.md

```diff
@@ -7,36 +7,34 @@
 
 This section shows how you can get help from the core dev team and the wider community.
 
-<a id="support"></a>
```

Comment:
Is this needed @hamishwillee ?

Reply:
Probably not. I add these sometimes to provide a short and consistent unchanging link. But not really needed.